blog/_layouts/default.html

```diff
@@ -0,0 +1,15 @@
+!DOCTYPE html>
```

Comment:
Hi, sorry it has taken so long to give this a technical review.

There is a small typo. I think this is missing a left bracket `<`.

`!DOCTYPE html>` `<!DOCTYPE html>`